app/src/main/java/com/example/alinagavrila/employees_administration/Company.java

```diff
@@ -1,15 +1,143 @@
 package com.example.alinagavrila.employees_administration;
 
+import com.example.alinagavrila.employees_administration.constants.Enums.Platform;
+import com.example.alinagavrila.employees_administration.constants.Enums.ProgrammingLanguage;
 import com.example.alinagavrila.employees_administration.constants.Enums.Gender;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import javax.xml.namespace.NamespaceContext;
 
 public class Company {
     private static List<Employee> employeeList = new ArrayList<>();
 
-    public static void addNewEmployee(Employee employee) {
-        employeeList.add(employee);
+    public static Employee newEmployee() throws IOException {
```

Comment:
this method should be renamed. it's not very intuitive what it is supposed to do